packages/scheduler/src/forks/SchedulerHostConfig.mock.js

```diff
@@ -94,6 +98,31 @@ export function unstable_flushNumberOfYields(count: number): void {
   }
 }
 
+export function unstable_flushUntilNextPaint(): void {
+  if (isFlushing) {
+    throw new Error('Already flushing work.');
+  }
+  if (scheduledCallback !== null) {
+    const cb = scheduledCallback;
+    shouldYieldForPaint = true;
```

Comment:
I supposed a better solution is to reassign `shouldYieldToHost`

algebraic_effects.gif